FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml

```diff
@@ -25,21 +25,23 @@
     <constant name="world_size" value="25*m"/>
     <constant name="world_x" value="world_size"/>
     <constant name="world_y" value="world_size"/>
-    <constant name="world_z" value="world_size"/>
+    <constant name="world_z" value="world_size"/>    
   </define>
 
   <include ref="./DectDimensions.xml" />
 
-  <include ref="Beampipe.xml"/>
+  <!--<include ref="Beampipe.xml"/>-->
+  <include ref="../../../MDI/compact/MDI_o1_v01/Beampipe_CADimport_o1_v02.xml" />
   <include ref="BeamInstrumentation.xml"/>
   <include ref="LumiCal.xml"/>
-  <include ref="HOMAbsorber.xml"/>
+  <!--<include ref="HOMAbsorber.xml"/>-->
   <include ref="Vertex_IDEA_o1_v01.xml"/> <!-- symbolic link to ../../../IDEA/compact/IDEA_o1_v03/Vertex_IDEA_o1_v01.xml"/>-->
   <include ref="DriftChamber_o1_v02.xml"/> <!-- symbolic link to ../../../IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml -->
   <include ref="ECalBarrel_thetamodulemerged.xml"/> <!-- if you remove the ECalBarrel, you also have to remove or update the "GlobalSolenoid" field (it depends on ECAL dimensions) -->
   <include ref="HCalBarrel_TileCal.xml"/>
   <include ref="ECalEndcaps_coneCryo.xml"/>
   <include ref="HCalEndcaps_ThreeParts_TileCal.xml"/>
+  <include ref="HCalBarrel_TileCal.xml"/>
```

Comment:
Twice is a bit much? (this already exists 3 lines before).

Maybe `git add --patch` ? Maybe merge conflict gone wrong?

Reply:
fixed